<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>855</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MolonaViz</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>3</y>
      <width>929</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="1">
      <widget class="QMdiArea" name="mdiArea"/>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QDockWidget" name="dockSensors">
         <property name="styleSheet">
          <string notr="true">
</string>
         </property>
         <property name="windowTitle">
          <string>Sensors</string>
         </property>
         <widget class="QWidget" name="dockWidgetContents">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QTreeView" name="treeViewSensors">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <attribute name="headerVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QDockWidget" name="dockPoint">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="windowTitle">
          <string>Points</string>
         </property>
         <widget class="QWidget" name="dockWidgetContents_2">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QTreeView" name="treeViewPoint">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <attribute name="headerVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1077</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStudy">
    <property name="title">
     <string>Study</string>
    </property>
    <addaction name="actionCreate_Study"/>
    <addaction name="actionOpen_Study"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuPoint">
    <property name="title">
     <string>Point</string>
    </property>
    <addaction name="actionImport_Point"/>
   </widget>
   <addaction name="menuStudy"/>
   <addaction name="menuPoint"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCreate_Study">
   <property name="text">
    <string>Create Study</string>
   </property>
  </action>
  <action name="actionOpen_Study">
   <property name="text">
    <string>Open Study</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionImport_Point">
   <property name="text">
    <string>Import Point </string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
